### PR TITLE
CI: don't run builds/tests when CHANGELOG.md or README.md were modified

### PR DESCRIPTION
**Pull request by rschu1ze**
Cf. https://github.com/ClickHouse/ClickHouse/pull/46764#issuecomment-1441698352

### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by rschu1ze** [comment]
@Felixoid Like to check? Thanks.

File: .github/workflows/docs_check.yml [hunk 1]
```diff
@@ -13,10 +13,12 @@ on:  # yamllint disable-line rule:truthy
     branches:
       - master
     paths:
+      - 'CHANGELOG.md'
+      - 'README.md'
```
Comment (Felixoid) [inline .github/workflows/docs_check.yml:17]:
```suggestion
      - 'README.md'
      - 'SECURITY.md'
```

Security file is kinda the same type

**Review by Felixoid — APPROVED** [review]
Good to go after some changes

File: .github/workflows/docs_check.yml [hunk 2]
```diff
@@ -13,10 +13,12 @@ on:  # yamllint disable-line rule:truthy
...
       - 'docker/docs/**'
       - 'docs/**'
-      - 'website/**'
       - 'utils/check-style/aspell-ignore/**'
+      - 'website/**'
```
[the remark this answers is not in the document]
Comment (rschu1ze) [reply .github/workflows/docs_check.yml:21]: Ah, missed that, tnx.

File: .github/workflows/docs_check.yml (hunk 1 above)
Comment (rschu1ze) [reply .github/workflows/docs_check.yml:17]: True

**Pull request merged**